run-test.sh

```diff
@@ -128,6 +128,12 @@ create_test_overlay()
 	exit 1
   fi
   find $CoreFxBins -name '*.dll' -exec cp '{}' "$OverlayDir" ";"
+
+  # Then the native linux CoreFX binaries
```

Comment:
We don't currently run our tests on OS X in CI, but hopefully we will soon, asn presumably this script will be used for that, too?